google-cloud-clients/google-cloud-compute/src/main/java/com/google/cloud/compute/Region.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.compute;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Enumeration of Regions.
+ *
+ * @see <a href="https://cloud.google.com/compute/docs/regions-zones/">Regions and Zones</a>
+ */
+public enum Region {
```

Comment:
Would you mind renaming this to `Regions`? Normally I think enums are named in the singular, but since there is already a Region resource class (in a different package sure), it might help with IDE autocomplete and readability etc if this is named differently than the autogenerated resource class.